lib/registries/addon/drafts/draft/-components/right-nav/styles.scss

```diff
@@ -5,3 +5,7 @@
         width: 100%;
     }
 }
+
+.SaveFailed {
```

Comment:
We seem to have another `.SaveFailed` class in `draft/style.scss` that is not used. Might as well remove that class in favor of this one here.